installing/installing_aws/installing-restricted-networks-aws-installer-provisioned.adoc

```diff
@@ -10,24 +10,24 @@ In {product-title} version {product-version}, you can install a cluster on Amazo
 == Prerequisites
 
+* You reviewed details about the xref:../../architecture/architecture-installation.adoc#architecture-installation[{product-title} installation and update] processes.
+* You read the documentation on xref:../../installing/installing-preparing.adoc#installing-preparing[selecting a cluster installation method and preparing it for users].
 * You xref:../../installing/installing-mirroring-installation-images.adoc#installation-about-mirror-registry_installing-mirroring-installation-images[mirrored the images for a disconnected installation] to your registry and obtained the `imageContentSources` data for your version of {product-title}.
 +
 [IMPORTANT]
 ====
 Because the installation media is on the mirror host, you can use that computer to complete all installation steps.
 ====
 * You have an existing VPC in AWS. When installing to a restricted network using installer-provisioned infrastructure, you cannot use the installer-provisioned VPC. You must use a user-provisioned VPC that satisfies one of the following requirements:
-** Contains the mirror registry.
-** Has firewall rules or a peering connection to access the mirror registry hosted elsewhere.
-* You reviewed details about the xref:../../architecture/architecture-installation.adoc#architecture-installation[{product-title} installation and update] processes.
+** Contains the mirror registry
+** Has firewall rules or a peering connection to access the mirror registry hosted elsewhere
 * You xref:../../installing/installing_aws/installing-aws-account.adoc#installing-aws-account[configured an AWS account] to host the cluster.
 +
 [IMPORTANT]
 ====
 If you have an AWS profile stored on your computer, it must not use a temporary session token that you generated while using a multi-factor authentication device. The cluster continues to use your current AWS credentials to create AWS resources for the entire life of the cluster, so you must use key-based, long-lived credentials. To generate appropriate keys, see link:https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_access-keys.html[Managing Access Keys for IAM Users] in the AWS documentation. You can supply the keys when you run the installation program.
 ====
-* You downloaded the AWS CLI and installed it on your computer. See
-link:https://docs.aws.amazon.com/cli/latest/userguide/install-bundle.html[Install the AWS CLI Using the Bundled Installer (Linux, macOS, or Unix)] in the AWS documentation.
+* You downloaded the AWS CLI and installed it on your computer. See link:https://docs.aws.amazon.com/cli/latest/userguide/install-bundle.html[Install the AWS CLI Using the Bundled Installer (Linux, macOS, or Unix)] in the AWS documentation.
 * If you use a firewall and plan to use the Telemetry service, you xref:../../installing/install_config/configuring-firewall.adoc#configuring-firewall[configured the firewall to allow the sites] that your cluster requires access to.
```

Comment:
I know this is existing text. All other instances of _telemetry_ use lower-case T except this and the following.

Reply:
Good catch. Uppercase "Telemetry" is used to refer to the Telemetry service that Red Hat provides, as detailed in the [Remote health monitoring](https://deploy-preview-33681--osdocs.netlify.app/openshift-enterprise/latest/support/remote_health_monitoring/about-remote-health-monitoring.html#understanding-telemetry-and-insights-operator-data-flow_about-remote-health-monitoring) section of the docs. Additionally, a number of the assemblies that include this prerequisite have instances of "the Telemetry service" elsewhere in the content.

I have subsequently replaced "telemetry" in each of the related prerequisites in this PR to refer to "the Telemetry service".